src/Files.App/Utils/Storage/Helpers/FilePropertiesHelpers.cs

```diff
@@ -120,7 +121,14 @@ public static void OpenPropertiesWindow(object item, IShellPage associatedInstan
 			appWindow.TitleBar.ButtonBackgroundColor = Colors.Transparent;
 			appWindow.TitleBar.ButtonInactiveBackgroundColor = Colors.Transparent;
 
-			appWindow.SetIcon(applicationService.AppIcoPath);
+			string appIcoPath = ApplicationService.AppEnvironment switch
```

Comment:
Doesn't the application service already contain this logic?

Reply:
That was marked as obsolete.

Reply:
The service or just that method?

Reply:
Just that.

Reply:
I think it's worth having it in a single place instead of duplicating it.

Reply:
Should I do that in this PR?

Reply:
Might as well 👍

Reply:
I decided to use AppLifecycleHelper to put those values. Thanks for your idea.